From 1d458688d31c29028f7efa8efc1502230f428b29 Mon Sep 17 00:00:00 2001
From: Jonathon Jongsma <jjongsma@redhat.com>
Date: Tue, 1 Jul 2014 15:10:48 -0500
Subject: [PATCH] remote-viewer: mention vv-file in help output

The man page already has a description of the vv-file format, but the
--help output didn't mention it how to use it.

References: rhbz#970825
---
 src/remote-viewer-main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/remote-viewer-main.c b/src/remote-viewer-main.c
index 505074e..5533098 100644
--- a/src/remote-viewer-main.c
+++ b/src/remote-viewer-main.c
@@ -104,7 +104,7 @@ main(int argc, char **argv)
           N_("Open connection using Spice controller communication"), NULL },
 #endif
         { G_OPTION_REMAINING, '\0', 0, G_OPTION_ARG_STRING_ARRAY, &args,
-          NULL, "-- URI" },
+          NULL, "URI|VV-FILE" },
         { NULL, 0, 0, G_OPTION_ARG_NONE, NULL, NULL, NULL }
     };
 
